python/GafferUI/ColorChooserPlugValueWidget.py

```diff
@@ -86,6 +86,22 @@ def __init__( self, plugs, **kw ) :
 		self.__lastChangedReason = None
 		self.__mergeGroupId = 0
 
+	def setInitialColor( self, color ) :
+
+		self.__colorChooser.setInitialColor( color )
+
+	def getInitialColor( self ) :
+
+		self.__colorChooser.getInitialColor()
```

Comment:
Are we missing a `return` here?

Reply:
Most definitely, fixed and squashed into https://github.com/GafferHQ/gaffer/pull/6106/commits/774eea8a6d4699c21f66024310cb6755fcf68217